packages/next-frontend/components/discourse.tsx

```diff
@@ -0,0 +1,57 @@
+import Card from "@material-ui/core/Card";
+import Box from "@material-ui/core/Box";
+import Typography from "@material-ui/core/Typography";
+import CardHeader from "@material-ui/core/CardHeader";
+import Avatar from "@material-ui/core/Avatar";
+import ReactMarkdown from "react-markdown/react-markdown.min";
+import GetPostResponse from "../types";
+import { useState } from "react";
+import { useEffect } from "react";
+import axios from "axios";
+
+const discourseServer = 'https://gov.near.org';
+
+interface DiscoursePostProps {
+  postId: string;
+}
+
+const DiscoursePost: React.FC<DiscoursePostProps> = (
+  props: DiscoursePostProps
+) => {
+  const [post, setPost] = useState({} as GetPostResponse);
+
+  useEffect(() => {
+    if (!post.id) {
+      axios
```

Comment:
I don't know what's the general agreement on dependency management, but I used `axios` in the leaderboard PR before switching to [`fetch`](https://github.com/HypeDAO/HypeDAO/pull/8/files#diff-f51529105cea8ba9387091f3e88cce1f6ddf214250a9627dd29e576c9424a115R75). My reasoning was to have less dependencies. @EV3RETH What's your take on this?

Reply:
I think the less dependancies the better as well

Reply:
I also like having less dependencies